assets/js/vue/img.js

```diff
@@ -1,13 +1,23 @@
-// This is a very basic img component and doesn't have the logic that the ruby version has.
-// we need to build that logic to implement a somewhat same version of that component
-// https://github.com/nhoizey/jekyll-cloudinary/blob/master/lib/jekyll/cloudinary.rb
-import Tools from '../tools.js';
 import Presets from '../presets.js';
+import TransformationOptions from '../transformation-options.js';
+import Cloudinary from '../../../.storybook/config/cloudinary.js';
 
 const basePath = 'https://res.cloudinary.com/c4a8/image/upload/';
 
 export default {
   tagName: 'v-img',
+  data() {
+    return {
+      dimensions: {
+        naturalWidth: null,
+        naturalHeight: null,
+      },
+      srcset: '',
+      cloudinary: Cloudinary,
```

Comment:
Hat es eigentlich einen Grund, warum du die globalen Importe auch in das vue objekt kopierst? Kannst du doch auch einfach über Cloudinary zugreifen.